identifiers/country-mx/states.csv

```diff
@@ -0,0 +1,34 @@
+id,name
+ocd-division/country:mx,Mexico
+ocd-division/country:mx/state:ag,Aguascalientes
+ocd-division/country:mx/state:bc,Baja California
+ocd-division/country:mx/state:bs,Baja California Sur
+ocd-division/country:mx/state:cm,Campeche
+ocd-division/country:mx/state:cs,Chiapas
+ocd-division/country:mx/state:ch,Chihuahua
+ocd-division/country:mx/state:co,Coahuila
+ocd-division/country:mx/state:cl,Colima
+ocd-division/country:mx/state:df,Mexico City
```

Comment:
'Mexico City' is listed as 'Ciudad de México' on INE site, would it be better to match?

Reply:
See above comment.